### PR TITLE
Fix typos in documentation.

### PR DESCRIPTION
**Pull request by ixjlyons**
Trivial fix but at least I'm able to contribute somehow :)

edit: Actually, I went ahead and checked the spelling on all of the top-level docs and caught a few more things. Would you prefer them all in one PR or separate ones for each file?

### CODE REVIEW

**Comment by jolange** [comment]
I guess one PR for doc typos should be fine.

**Comment by ixjlyons** [comment]
Ok, I edited the title to reflect more general changes and squashed down to one commit.

File: docs/userguide [hunk 1]
```diff
@@ -1423,7 +1423,7 @@ tray_padding <px> [px]
 
 *Example*:
 -------------------------
-# Obey Fitts's law
+# Obey Fitts' law
```
Comment (jolange) [inline docs/userguide]: Actually, Wikipedia says
> Fitts's law (often cited as Fitts' law) is...

Comment (ixjlyons) [inline docs/userguide]: Interesting. I actually do research related to human computer interfaces and I've never seen it written as `Fitts's law` (I've seen it written as `Fitt's law`, but that's definitely incorrect). There doesn't seem to be a definitive consensus one way or the other regarding possessive form of proper names ending in "s" [[1]], but I'm in favor of fewer changes.

[1]: http://www.dailywritingtips.com/possessive-of-proper-names-ending-in-s/

**Comment by ixjlyons** [comment]
I don't think there are any controversial changes remaining :) Should be good to go.

**Pull request merged**